tests/publiccloud/patch_and_reboot.pm

```diff
@@ -27,6 +27,13 @@ sub run {
     my $remote = $args->{my_instance}->username . '@' . $args->{my_instance}->public_ip;
     # pkcon not present on SLE-micro
     kill_packagekit($args->{my_instance}) unless (is_sle_micro);
+
+    # Record package list before fully patch system
+    if (get_var('SAVE_LIST_OF_PACKAGES')) {
```

Comment:
I think that this test module is only relevant for mr-test and not for HanaSR

Reply:
Yes, it's only for mr-test.